### ISSUE TITLE
5.0.1 error on Android

### ISSUE THREAD

**Issue by billnbell** [react-native-cameraroll/react-native-cameraroll#421]
Getting error compiling - no exception error:

@react-native-camera-roll/camera-roll/android/src/main/java/com/reactnativecommunity/cameraroll/CameraRollModule.java

2 spots:

```
-      try {
-        retriever.release();
-      } catch (IOException e) {
-        // Do nothing. We can't handle this, and this is usually a system problem
-      }
+      retriever.release();
```

**Comment by Romick2005** [comment]
https://github.com/react-native-cameraroll/react-native-cameraroll/commit/a0abf74c2e5d10057c93725c70c252ffc56d4f33

**Comment by billnbell** [comment]
Well it breaks on 31... Can we fix it please?

**Comment by hit-cod** [comment]
same error, any progress?

**Comment by Arjan-Zuidema** [comment]
I am looking into it right now

**Comment by Romick2005** [comment]
I suggest to catch wider exceptions:
```
       try {
         retriever.release();
-      } catch (IOException e) {
+      } catch (Exception e) {
         // Do nothing. We can't handle this, and this is usually a system problem
       }
```

**Comment by itog** [comment]
Raising the Sdkversion to 33 gives the same error in v4.1.2

**Comment by Arjan-Zuidema** [comment]
This should be fixed in 5.0.2
